doc/source/dev/index.rst

```diff
@@ -2,6 +2,31 @@
 Contributing to NumPy
 #####################
 
+Not a coder? Not a problem! NumPy is multi-faceted, and we can use a lot of help.
+These are all activities we'd like to get help with (they're all important, so
+we list them in alphabetical order):
+
+- Code maintenance and development
+- Community coordination
+- DevOps
+- Developing educational content
+- Fundraising
+- Project management
+- Marketing
+- Translating content
+- Website design and development
+
+The rest of this document discusses working on the NumPy code base and documentation.
+We're in the process of updating our descriptions of other activities and roles.
+If you are interested in these other activities, please contact us!
+You can do this via
+the [numpy-discussion mailing list](https://scipy.org/scipylib/mailing-lists.html)),
+or on GitHub (open an issue or comment on a relevant issue). These are our preferred
+communication channels (open source is open by nature!), however if you prefer
+to discuss in private first, please reach out to our community coordinators
+at (TODO: email address that's linked to a Google Group, or an individual address?).
```

Comment:
Is anyone interested in / willing to be contacted in a non-public way. We could do a couple of things here:

- a Google Group with a descriptive name
- directly add one or more individual email addresses
- create a role, e.g. "community coordinator" or "project manager", and then give that role to one or more people

I don't have a strong preference, just wondering what others think.

Reply:
Discussed at meeting today: thinking was that Slack or similar forum which allows both open and closed conversation may be helpful.  We can identify a pool of mentors to reach out to.

Reply:
I think we said both Slack and a private email address. I'll set up both.

Reply:
- created `numpy-team@googlegroups.com`. 
- sent email to mailing list about `numpy.slack.com`; it exists but can't tell who created it.

Reply:
`numpy.slack.com` no response, created `numpy-team.slack.com`. nicely symmetric:)